.github/workflows/flutter-build-android.yml

```diff
@@ -44,66 +44,66 @@ jobs:
   build:
     runs-on: macos-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-java@v3
-      with:
-        distribution: 'temurin' # See 'Supported distributions' for available optionsjava-version
-        java-version: '17'
-    - name: Install the Apple certificate and provisioning profile
-      env:
-        BUILD_CERTIFICATE_BASE64: ${{ secrets.ios_signing_key_asc }}
-        P12_PASSWORD: ${{ secrets.ios_signing_key_password }}
-        BUILD_PROVISION_PROFILE_BASE64: ${{ secrets.ios_provisioning_profile_asc }}
-        KEYCHAIN_PASSWORD: ${{ secrets.ios_keychain_password }}
-      run: |
-        # create variables
-        CERTIFICATE_PATH=$RUNNER_TEMP/build_certificate.p12
-        PP_PATH=$RUNNER_TEMP/build_pp.mobileprovision
-        KEYCHAIN_PATH=$RUNNER_TEMP/app-signing.keychain-db
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin' # See 'Supported distributions' for available optionsjava-version
+          java-version: '17'
```

Comment:
is het verstandig om deze zo vast te zetten? Is het niet beter gewoon de latest stable te pakken?

Reply:
Changes in flutter_build-android.yml zijn alleen door formatting.
Daarnaast werkt dit sowieso niet goed voor de manier waarop wij dit doen. Latest stable geeft gradle errors, dus ik heb mijn eigen versie lager gezet dan 17

Reply:
Volgens mij de de install voor JAVA ook helemaal niet nodig, ik gebruik dit ook niet voor de geolocator en permission_handler plugins en ook de base_app gebruikt het niet.